addon/model.js

```diff
@@ -450,7 +390,7 @@ export default class MegamorphicModel extends Ember.Object {
   }
 
   _changedKeys(data) {
```

Comment:
I know this was pre-existing but this method name is unclear, perhaps `_updateChangedKeys` would be better.

Reply:
This maps to the `InternalModel._changedKeys`. Apart from this, I don't have preference for the name.